corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -407,6 +437,40 @@ def delete_all_cases(domain=None):
             cursor.execute('SELECT delete_all_cases(%s)', [domain])
 
     @staticmethod
+    def get_all_cases_modified_since(server_modified_on_since=None, chunk_size=500):
```

Comment:
seems like this could be abstracted into the same function as forms, but you likely have more context if that's possible